test/modules/TwinTester/StatusCode.cs

```diff
@@ -8,10 +8,12 @@ enum StatusCode
         DesiredPropertyReceived = 202,
         ReportedPropertyReceived = 203,
         ReportedPropertyUpdated = 204,
+        ReportedPropertyReceivedAfterThreshold = 205,
```

Comment:
Seems weird to me that this is a success, considering the update didn't come in time. If you want to separate this from the errors I think it is good to use another leading number (i.e. 3 or 4)

Reply:
I am open to use "5". 😃